src/elements/common/nav-button/NavButton.js

```diff
@@ -48,6 +48,7 @@ const NavButton = React.forwardRef<Props, React.Ref<any>>((props: Props, ref: Re
                 return (
                     <Component
                         className={classNames(className, { [activeClassName]: isActiveValue })}
+                        getDOMRef={ref}
```

Comment:
This will only work for certain component types, like PlainButton, but not for native elements. Is this necessary with the `{...rest}` below?